src/collection/class.js

```diff
@@ -72,7 +67,7 @@ let elesfn = ({
 
   hasClass: function( className ){
     let ele = this[0];
-    return ( ele != null && ele._private.classes[ className ] ) ? true : false;
+    return ( ele != null && ele._private.classes.has(className) ) ? true : false;
```

Comment:
Your change looks fine on this line, but `? true : false` could be removed to make it shorter.